packages/front-end/components/BillingErrorBanner.tsx

```diff
@@ -0,0 +1,58 @@
+import React, { useState } from "react";
```

Comment:
We already have some logic in `TopNav.tsx` to show an error when the status is past due.